pom.xml

```diff
@@ -56,21 +56,21 @@
 
   <properties>
     <surefire.version>2.21.0</surefire.version>
-    <mavenVersion>3.0</mavenVersion>
-    <javaVersion>7</javaVersion>
+    <mavenVersion>3.2.5</mavenVersion>
```

Comment:
In will be good to have changing Maven version and JDK requirement  in separate change with separate issue.

Reply:
I can expand the issue, but it's not really possible. Changing the parent pom required changing a bunch of other things.

Reply:
java version upgrade must be separate issue